app/controllers/course/courses_controller.rb

```diff
@@ -8,17 +8,12 @@ def index
   end
 
   def show
-    respond_to do |format|
-      format.json do
```

Comment:
Why are you removing the `respond_to` block?

Reply:
I remembered there was this discussion in the past (I forgot exactly when) that we need to try getting rid of all the `respond_to` block in our backend.. The reasoning being that this block serves the way to forward the output to frontend if there are possibilities of different type of request being made from FE (in the past, we still utilise HTML as well as JSON format request)

-> and therefore usually for this `respond_to` block, it will differentiate between `format.json` process as well as `format.html`

Now, since all of the request from FE has JSON format, this should not be used anymore.